### PR TITLE
[protobuf] Update protobuf to v3.11.4

### PR DESCRIPTION
**Pull request by cqjjjzr**
Specify library name and version:  **protobuf/3.11.4**

- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by cqjjjzr** [comment]
@uilianries Then how can I specify different test_package data and .patch file for each version?

**Comment by cqjjjzr** [comment]
I just modified the script to select the proper file for each version. Could you please give it a look?

File: recipes/protobuf/all/test_package/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        if os.path.exists(os.path.join(self.source_folder, "addressbook.pb.cc")):
+            os.remove(os.path.join(self.source_folder, "addressbook.pb.cc"))
+        if os.path.exists(os.path.join(self.source_folder, "addressbook.pb.h")):
+            os.remove(os.path.join(self.source_folder, "addressbook.pb.h"))
```
Comment (danimtb) [inline recipes/protobuf/all/test_package/conanfile.py:14]: Is this needed?

Comment (uilianries) [reply recipes/protobuf/all/test_package/conanfile.py:14]: I didn't understand it also.

**Comment by uilianries** [comment]
@cqjjjzr Now you have some conflicts, could you please check them?

File: recipes/protobuf/all/test_package/conanfile.py (hunk 1 above)
Comment (cqjjjzr) [reply recipes/protobuf/all/test_package/conanfile.py:14]: This is to delete existing generated code, which can force compiler to re-compile these source files. Actually I can barely remember the problem leading to this fix.

**Comment by cqjjjzr** [comment]
Additionally, I found protobuf 3.11.4 already included in the latest branch. Maybe we can directly close this PR.

**Comment by Croydon** [comment]
Protobuf 3.11.4 got added via https://github.com/conan-io/conan-center-index/pull/1199

**Pull request closed without merging**